mlir-sycl/lib/Conversion/SYCLToLLVM/SYCLToLLVM.cpp

```diff
@@ -1056,6 +1069,36 @@ class ConstructorPattern final
   }
 };
 
+//===----------------------------------------------------------------------===//
+// AccessorGetPointerPattern - Convert `sycl.accessor.get_pointer` to LLVM.
+//===----------------------------------------------------------------------===//
+
+class AccessorGetPointerPattern
+    : public LoadMemberDimPattern<SYCLAccessorGetPointerOp,
+                                  AccessorGetMultiPtr> {
+public:
+  using LoadMemberDimPattern<SYCLAccessorGetPointerOp,
+                             AccessorGetMultiPtr>::LoadMemberDimPattern;
+};
+
+//===----------------------------------------------------------------------===//
+// AccessorSizePattern - Convert `sycl.accessor.size` to LLVM.
+//===----------------------------------------------------------------------===//
+
+class AccessorSizePattern
+    : public GetRangeSizePattern<SYCLAccessorSizeOp>,
+      public GetMemberPattern<AccessorGetMemRange, RangeGetDim> {
```

Comment:
Id' say here you need to use a different tag, as you'll be accessing a different range in the accessor ([`MAccessRange`](https://github.com/intel/llvm/blob/c39de7135bcd1629fa09a0f15a24ab7c23a85516/sycl/source/accessor.cpp#L57) instead of `MMemoryRange`)